cluster/directory/base_directory.go

```diff
@@ -24,13 +24,22 @@ import (
 	"go.uber.org/atomic"
 )
 import (
+	"github.com/apache/dubbo-go/cluster"
 	"github.com/apache/dubbo-go/common"
+	"github.com/apache/dubbo-go/common/constant"
+	"github.com/apache/dubbo-go/common/extension"
+	"github.com/apache/dubbo-go/common/logger"
+	gxset "github.com/dubbogo/gost/container/set"
 )
 
+var RouterUrlSet = gxset.NewSet()
```

Comment:
pls attention that, the proper noun like ID/URL/HTTP, either using them like "id/url/http" or "ID/URL/HTTP". u should not use them like "Id/Url/Http".

Reply:
i see ,Hound ci will Tips this